test/3DPrinting.js

```diff
@@ -0,0 +1,73 @@
+process.env.NODE_ENV = 'test';
+
+const chai = require('chai');
+const chaiHttp = require('chai-http');
+const constants = require('../api/constants');
+const { OK, BAD_REQUEST } = constants.STATUS_CODES;
+const tools = require('../util/testing-utils/tools.js');
+const send3dPrintRequest
+= require('../api/printingRPC/client/printing3d/print_3d_client');
+const sinon = require('sinon');
+
+let app = null;
+const expect = chai.expect;
+
+chai.should();
+chai.use(chaiHttp);
+const SUCCESS_MESSAGE = 'success';
+const ERROR_MESSAGE = 'error';
+const TEXT_REQUEST = {
+  raw: '',
```

Comment:
why not put something as a value here?

Reply:
I based it off of jerry's test file, he didn't put anything there. Should I change it?

Reply:
Lawrence is correct. We don't need a value. I'm assuming the raw values of 3D printing files will be massive. To pass in the raw data of a test print would take up too much space. We are testing the ability for the RPC to be executed, not whether it can print a file or not. Thus, an empty string would be fine.